### PR TITLE
Adding support to `lessOptions`

### PR DESCRIPTION
**Pull request by gullitmiranda**

### CODE REVIEW

File: src/visitors/transpileLess.js [hunk 1]
```diff
@@ -28,8 +28,18 @@ const transpile = (less, source, filename, opts = {}) => {
             break;
     }
 
-    const parse = deasync((input, options, callback) => less.parse(input, options, (e, root, imports, options) => callback(e, {root, imports, options})));
-    const {root, imports, options} = parse(source, {math: 0, paths, banner});
+    const parse = deasync((input, options, callback) =>
+      less.parse(input, options, (e, root, imports, options) =>
+        callback(e, {root, imports, options})));
+
+    const parseOpts = Object.assign(
```
[the remark this answers is not in the document]
Comment (gullitmiranda) [reply src/visitors/transpileLess.js]: me too, but babel is not configured to support the object spread:
```
SyntaxError: src/visitors/transpileLess.js: Unexpected token (36:48)
  34 |         callback(e, {root, imports, options})));
  35 |
> 36 |     const parseOpts = { math: 0, paths, banner, ...opts.lessOptions };
     |                                                 ^
  37 |     const { root, imports, options } = parse(source, parseOpts);
  38 |
  39 |     if (!root) {
```

**Comment by jean343** [comment]
Thank you so much for the PR, I really appreciate!
I like the example and test code, it makes the JS code clean.

If it's not too much work, could you reduce the size of the sample and test by importing only the necessary functions, I don't think that we need the entire `tinycolor.less` file to support the test code.

Thanks,

File: examples/.babelrc [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "plugins": [
-    "styless",
+    ["../dist", { "lessOptions": { "javascriptEnabled": true } }],
```
Comment (jean343) [inline examples/.babelrc]: I understand that it is easier to test this way, but I prefer importing `styless` to test the npm package.

**Comment by gullitmiranda** [comment]
It's not a problem, I will do when got back to my computer.

Gullit Miranda

Em dom, 25 de nov de 2018 20:59, Jean-Philippe Bergeron <
notifications@github.com escreveu:

**Comment by gullitmiranda** [comment]
so the settings I made will only work after the package is published, which
isn't a problem for me

Gullit Miranda

Em dom, 25 de nov de 2018 21:00, Jean-Philippe Bergeron <
notifications@github.com escreveu:

File: examples/.babelrc (hunk 2 above)
Comment (gullitmiranda) [reply examples/.babelrc]: done

**Comment by gullitmiranda** [comment]
@jean343, updated

**Pull request merged**